spec/acceptance/overridden_settings_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'spec_helper_acceptance'
+
+# These tests are designed to ensure that the module, when ran overrides,
+# sets up everything correctly and allows us to connect to Postgres.
+describe 'postgresql::server', unless: UNSUPPORTED_PLATFORMS.include?(fact('osfamily')) do
+  pp = <<-MANIFEST
+    class { 'postgresql::server':
+      roles          => {
+        'testusername' => {
+          password_hash => postgresql_password('testusername', 'supersecret'),
+          createdb      => true,
+        },
+      },
+      config_entries => {
+        max_connections => 200,
```

Comment:
This seems to be causing issues on some of our test nodes:
```
09:04:41   Apr 03 09:04:39 k8ytf5mtkdzr708 pg_ctl[14776]: FATAL:  could not create shared memory segment: Invalid argument
09:04:41   Apr 03 09:04:39 k8ytf5mtkdzr708 pg_ctl[14776]: DETAIL:  Failed system call was shmget(key=5432001, size=36954112, 03600).
09:04:41   Apr 03 09:04:39 k8ytf5mtkdzr708 pg_ctl[14776]: HINT:  This error usually means that PostgreSQL's request for a shared memory segment exceeded your kernel's SHMMAX parameter.  You can either reduce the request size or reconfigure the kernel with larger SHMMAX.  To reduce the request size (currently 36954112 bytes), reduce PostgreSQL's shared memory usage, perhaps by reducing shared_buffers or max_connections.
09:04:41   Apr 03 09:04:39 k8ytf5mtkdzr708 pg_ctl[14776]: If the request size is already small, it's possible that it is less than your kernel's SHMMIN parameter, in which case raising the request size or reconfiguring SHMMIN is called for.
```

Reply:
Should I lower it to 20?